src/config_files.rs

```diff
@@ -71,40 +71,29 @@ pub(crate) fn read_config_file(
             };
 
             match answer.to_lowercase().trim() {
-                "y" | "" => {
-                    let mut output = File::create(&config_path).expect("Unable to create file");
-                    write!(output, "{}", config_file).expect("Unable to write to config file");
-                    println!("Config file created at: {}", config_path.to_string_lossy());
-                }
-                _ => {
-                    println!("Continuing without config file");
-                    // Just use the contents of "default_config.nu" or "default_env.nu"
-                    eval_source(
-                        engine_state,
-                        stack,
-                        config_file.as_bytes(),
-                        if is_env_config {
-                            "default_env.nu"
-                        } else {
-                            "default_config.nu"
-                        },
-                        PipelineData::new(Span::new(0, 0)),
-                    );
-
-                    // Merge the environment in case env vars changed in the config
-                    match nu_engine::env::current_dir(engine_state, stack) {
-                        Ok(cwd) => {
-                            if let Err(e) = engine_state.merge_env(stack, cwd) {
-                                let working_set = StateWorkingSet::new(engine_state);
-                                report_error(&working_set, &e);
-                            }
+                "y" | "" => match File::create(&config_path) {
+                    Ok(mut output) => match write!(output, "{}", config_file) {
+                        Ok(_) => {
+                            println!("Config file created at: {}", config_path.to_string_lossy())
                         }
                         Err(e) => {
-                            let working_set = StateWorkingSet::new(engine_state);
-                            report_error(&working_set, &e);
+                            eprintln!(
+                                "Unable to write to {}: {}",
+                                config_path.to_string_lossy(),
+                                e
+                            );
+                            eval_default_config(engine_state, stack, config_file, is_env_config);
+                            return;
                         }
+                    },
+                    Err(e) => {
+                        eprintln!("Unable to create {}: {}", config_file, e);
```

Comment:
Can we say something like `Unable to create {}, sourcing default file instead.", just to let the user know that something is being read?  I think that's what's happening here.

Reply:
`eval_default_config` says "Continuing without config file", shall we change it to "Source default config instead"?

Reply:
as long as we still have a way to launch nushell without any config sourced at all, I think it's fine to do this type of thing with config and env.